### DISCUSSION TITLE
潮流周刊第179期 - 喜贡爆炒

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#145] [General]
# 潮流周刊第179期 - 喜贡爆炒

封面图拍摄来源于周末喜贡吃饭拍的店内图，假如你在杭州想去吃炒菜，辣的，下饭的，爆炒的那种，一定推荐去试试喜贡爆炒，可称杭州笨萝卜，6 分钟爆炒 3 个菜的节奏，上来时还有锅气的那种，价格 OK，很费米饭，很香很好吃，有一个同事可以吃 3 碗饭的那种。

https://weekly.tw93.fun/posts/179-%E5%96%9C%E8%B4%A1%E7%88%86%E7%82%92

**Comment by ChengWeiDong** [comment]
老师有木有粉丝群，拉一拉

**Comment by rennsax** [comment]
一份大约 30 篇 AI 研究论文的阅读清单那里，Arc 的分享链接好像打不开了，报错 Failed to add shared object to your sidebar，不知道为什么。要不还是列一下链接形式或者用其他兼容性更好的格式？感谢！

**Reply by tw93** [reply]
我点击了下有的，你可以继续试试
